model/src/main/proto/interaction_object.proto

```diff
@@ -53,5 +54,30 @@ message Fraction {
 
 // Structure for a ListOfString object.
 message ListOfSetsOfHtmlStrings {
-  repeated StringList set_of_html_strings = 1;
+  StringList set_of_html_strings = 1;
+}
+
+// Structure for a ImageWithRegions object.
+message ImageWithRegions {
+  string image_path = 1;
+  repeated LabelRegions label_regions = 2;
+}
+
+message LabelRegions {
+  string label = 1;
```

Comment:
Please add documentation for this & the next property to provide context. Also for the message itself.

Ditto for all other new messages & properties--comments help provide context. Consider that readers might not have any context on what these structures are for, so I suggest adding additional details other than just repeating the name (since that doesn't help provide information on _how_ to use the particular structure or define a particular property).

Reply:
done